ivre/passive.py

```diff
@@ -477,6 +477,17 @@ def _getinfos_ntlm(spec):
         if v
     }}
 
+def _getinfos_smb(spec):
```

Comment:
```suggestion

def _getinfos_smb(spec):
```
https://travis-ci.org/github/cea-sec/ivre/jobs/733330335